packages/test/test-gc-sweep-tests/src/channelHandler.ts

```diff
@@ -0,0 +1,234 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "assert";
+import { SharedCell } from "@fluidframework/cell";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+import { SharedCounter } from "@fluidframework/counter";
+import { IChannel } from "@fluidframework/datastore-definitions";
+import { Ink, IPen, IColor, IInkPoint } from "@fluidframework/ink";
+import { SharedMap } from "@fluidframework/map";
+import { ConsensusQueue, ConsensusResult } from "@fluidframework/ordered-collection";
+import { ConsensusRegisterCollection } from "@fluidframework/register-collection";
+
+export interface IRandomOpHandler {
+	executeRandomNonHandleOp(): Promise<void>;
+}
+
+/**
+ * TODO: opHandlers to add - SharedDirectoryHandler, SharedMatrixHandler and SharedStringHandler
+ * Not sure if SharedString can store and remove handles.
+ */
+export abstract class OpHandler {
+    protected readonly actions: (() => Promise<void>)[] = [];
+	private _random: Random | undefined;
+
+	public get random(): Random {
+		assert(this._random !== undefined, "Random needs to be set first before we do any op handling");
```

Comment:
what is the guideline for spaces vs tabs ? It seems we have all here :)

Reply:
4 spaces, I don't know why my editor started making tabs.